docs/QSG/setup_suse.md

```diff
@@ -343,19 +340,26 @@ Examples are available on [github](https://github.com/daos-stack/daos/tree/relea
 			log_file: /tmp/daos_engine_1.log
 			env_vars:
 				- CRT_TIMEOUT=30
-			scm_mount: /mnt/daos1
-			scm_class: dcpm
-			scm_list: [/dev/pmem1]
-			bdev_class: nvme
-			bdev_list: ["0000:83:00.0"]  # generate regular nvme.conf
+			storage:
+			-
+				class: dcpm
+				scm_mount: /mnt/pmem1
```

Comment:
should this scm_mount value be /mnt/daos1 instead?

Reply:
thanks for catching - fixed